test/test_utils.py

```diff
@@ -187,18 +193,21 @@ def test_upload(single_upload):
 
 
 def test_single_build_only(single_build):
-    assert os.path.exists(single_build)
+    for pkg in single_build:
+        assert os.path.exists(pkg)
 
 
 @pytest.mark.skipif(SKIP_DOCKER_TESTS, reason='skipping on osx')
 def test_single_build_with_post_test(single_build):
-    pkg_test.test_package(single_build)
+    for pkg in single_build:
+        pkg_test.test_package(pkg)
 
 
 @pytest.mark.long_running
 def test_multi_build(multi_build):
     for v in multi_build.values():
-        assert os.path.exists(v)
+        for pkg in v:
+            assert os.path.exists(pkg)
```

Comment:
I assume this test currently fails because of https://github.com/bioconda/bioconda-utils/pull/293/commits/a6dabe3cdc470e0c2c1c6ddd8a9e52a941e95589.